src/_support/Page/Acceptance/Administrator/AccessLevelControlFormPage.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+     * @package     Joomla.Test
+     * @subpackage  AcceptanceTester.Page
+     *
+     * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+     * @license     GNU General Public License version 2 or later; see LICENSE.txt
+     */
+namespace Page\Acceptance\Administrator;
+
+/**
+ * Acceptance Page object class to Access Level Control form page.
+ *
+ * @category  Users
+ * @package   Page\Acceptance\Administrator
+ * @copyright 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license   GNU General Public License version 2 or later; see LICENSE.txt
+ * @since     4.0.0
+ */
+class AccessLevelControlFormPage
+{
+    /**
+     * Url
+     *
+     * @var    string
+     * @since  4.0.0
+     */
+    public static $url = 'administrator/index.php?option=com_users&view=user&layout=edit';
+
+    /**
+     * Drop Down Toggle Element.
+     *
+     * @var    string
+     * @since  4.0.0
+     */
+    public static $dropDownToggle = "//button[contains(@class, 'dropdown-toggle')]";
+
+    /**
+     * Group Manager
+     *
+     * @var    string
+     * @since  4.0.0
+     */
+    public static $groupManager = '#1group_6';
+
+    /**
+     * Group Author
+     *
+     * @var    string
+     * @since  4.0.0
+     */
+    public static $groupAuthor = '#1group_3';
+
+    /**
+     * Manager Create
+     *
+     * @var    string
+     * @since  4.0.0
+     */
+    public static $managerCreateSelect = '#jform_rules_core.create_6';
```

Comment:
the selector seems a bit odd, the id of the element is like this?

Reply:
yes @puneet0191